### ISSUE TITLE
hakyll-4.15.0.0 testsuite failed

### ISSUE THREAD

**Issue by juhp** [jaspervdj/hakyll#887]
Stackage Nightly failed with the hakyll-4.15.0.0 testsuite:

```
      Hakyll.Core.Runtime.Tests                                                                                                   
        [ 1] run:              OK                                                                                                 
        [ 2] run:              OK                                                                                                 
        [ 3] run:                [ERROR] Hakyll.Core.Runtime.pickAndChase: Dependency cycle detected: partial.html.out1 depends on
 [(partial.html.out2,"_final")], partial.html.out2 depends on [(partial.html.out1,"_final")]                                      
    OK                                                                                                                            
        [ 4] run:                [ERROR] Hakyll.Core.Runtime.pickAndChase: Dependency cycle detected: partial.html.out1 depends on
 [(partial.html.out2,"raw")], partial.html.out2 depends on [(partial.html.out1,"raw")]                                            
    OK                                                                                                                            
        [ 5] run:                [ERROR] Hakyll.Core.Compiler.cached: You are trying to (perhaps indirectly) use `cached` on a non
-existing resource: there is no file backing template-empty.html                                                                  
    FAIL                                                                                                                          
          tests/Hakyll/Core/Runtime/Tests.hs:191:                                                                                 
          expected: ExitSuccess                                                                                                   
           but got: ExitFailure 1                                                                                                 
          Use -p '/Hakyll.Core.Runtime.Tests.[ 5] run/' to rerun this test only.                                                  
        [ 6] run:              OK                                                                                                 
```

**Comment by sternenseemann** [comment]
We are hitting the same issue in nixpkgs.

**Comment by Minoru** [comment]
Forgot to add the file to the Cabal package; sorry about that! I just published Hakyll 4.15.0.1 which fixes this issue.

I'll now work on a CI job that would catch such errors in the future, and will close this issue once that's done.

Thanks for reporting this, and sorry for the busywork.

**Comment by juhp** [comment]
Thanks!
